web/packages/teleterm/src/ui/components/CliCommand.tsx

```diff
@@ -57,12 +60,13 @@ export function CliCommand({
       justifyContent="space-between"
       borderRadius={2}
       bg="bgTerminal"
-      mb={2}
```

Comment:
Removing this margin didn't affect the layout in `OnlineDocumentGateway` because it was collapsed with the margin of the next block.